generate_network.py

```diff
@@ -4,79 +4,68 @@
 import networkx as nx
 import random, math
 import json
-import argparse, os, sys
+import sys, os
+import string
+import typer
+from enum import Enum
+
+# Consts
+class nw_types(Enum):
+    configmodel = "CONFIGMODEL"
+    scalefree = "SCALEFREE"                     # power law
+    newmanwattsstrogatz = "NEWMANWATTSSTROGATZ" # mesh, smallworld
+    barbell = "BARBELL"                         # partition
+    balancedtree = "BALANCEDTREE"               # committees?
+    star = "STAR"                               # spof
```

Comment:
This usually are uppercase as well

Reply:
the enums are uppercase as well?! wouldn't it look tad crowded?